Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.4
-MAINTAINER Michael de Wit <michael@drillster.com>
+FROM alpine:latest
+MAINTAINER Jesse Quinn <me@jessequinn.info>
```

Comment:
I love the work you've done for this project, but does that suddenly make you the maintainer? 😅

Reply:
returned to original name.